dj_database_url.py

```diff
@@ -102,14 +102,14 @@ def parse(url, engine=None, conn_max_age=0, ssl_require=False):
 
     # Handle postgres percent-encoded paths.
     hostname = url.hostname or ''
-    if '%2f' in hostname.lower():
+    if any(urldecode_char in hostname.lower() for urldecode_char in ['%2f', '%2a']):
```

Comment:
```suggestion
    if any(urldecode_char in hostname.lower() for urldecode_char in ['%2f', '%3a']):
```

Reply:
@dachia can you review and merge this suggestion, or reject it? I think it's right but I'd like your eyes on it too.